src/AppBundle/Api/EventProvider.php

```diff
@@ -38,6 +38,20 @@ public function getUpcomingEvents(): array
                     'lng' => (float) $event->getLongitude(),
                 ],
             ];
+
+            if ($committee = $event->getCommittee()) {
+                $el += [
+                    'committee_name' => $committee->getName(),
+                    'committee_url' => $this->urlGenerator->generate('app_committee_show', [
```

Comment:
You could instead inject the `CommitteeUrlGenerator` which has a dedicated method for generating a path by passing a committee object.